### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by djo1975**
Review one of your past projects
Learning objectives
Apply best practices in HTML code.
Use semantic HTML tags.
Use CSS selectors correctly.
Use CSS box model.
Perform a code review for yourself on a past project.
Estimated time: 0.5h
Description
In the previous lessons, you got an introduction to the code review process and you upgraded your HTML and CSS skills. That means that you are ready to perform a code review on your code!

IMPORTANT NOTE: Read all requirements before you start building your project.

Project requirements
Create a new empty repo.
Copy your code from the project that you completed during the Admissions Trial: - If you do not have the code from the Admissions Trial ask someone from your Standup Team to share their code with you. In that case, remember to add an attribution note and acknowledgment in your README file.
Create a new branch for your copied code.
Open a pull request.
Play the role of the Code Reviewer:
Follow the list from the article ["How to give a code review?"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).
Check if you have used semantic HTML tags (see examples of these tags in the first bullet of the best practices document below).
Check if you applied [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Add as many comments as possible in the given time and finish your review. Remember to add a minimum of 2 comments - you will definitely find something that could have been done better!
Copy a link to the pull request and ask for a code review in your Student Dashboard as usual. (The only difference will be that the Code Reviewer won't check your code but the review you performed - isn't it a code review inception? ;))
Work and submission mode
You should submit this activity individually.
Code review
Follow [these steps](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md) to request a code review of your project.

Submit your project
After the final approval from a code reviewer, you need to submit your project. [Read this FAQ for a reminder on how to submit your project.](https://microverse.zendesk.com/hc/en-us/articles/360061344234) Now go to your Student Dashboard and submit your project.

If you spot any bugs or issues in this activity, you can [open an issue with your proposed change](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/open_issue.md).

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    position:absolute;
+    left: 0px;
+    top: 312px;
+}
+@media screen and (min-width: 600px) {
+    body {
+      background-color: white;
+      font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    }
+  }
```
Comment (djo1975) [inline css/style.css:11]: I can run the program on my machine but your style is look very messy. I suggest you to change the position of you hamburger many.

File: README.md [hunk 2]
```diff
@@ -1,92 +1,5 @@
...
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page]([https://github.com/ArthurGC/portfolio-microverse/issues](https://github.com/djo1975/Portfolio-setup-and-mobile-first/issues)).
-
-## Acknowledgments
-
-- Thanks to Microverse
-- Thanks to @yodit93.
-## ?? License
-
```
Comment (djo1975) [inline README.md]: You have to implement linters.yml. to /.github)/workflows/ folder. If you need help you can fallow this link https://github.com/microverseinc/linters-config

**Review by djo1975 — COMMENTED** [review]
Hi, yodit93 you have to implement linters.yml. to /.github)/workflows/ folder before we continue.

**Comment by djo1975** [comment]
Description
In the previous lessons, you got an introduction to the code review process and you upgraded your HTML and CSS skills. That means that you are ready to perform a code review on your code!

**Comment by andgarzonmal** [comment]
Hi @djo1975, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**